src/Text/Pandoc/Readers/JATS.hs

```diff
@@ -370,36 +386,53 @@ parseBlock (Elem e) = do
          parseElement = filterChildren isEntry
          wrapWithHeader n mBlocks = do
                       isBook <- gets jatsBook
-                      let n' = if isBook || n == 0 then n + 1 else n
+                      let n' = case (filterChild (named "title") e >>= maybeAttrValue "display-as") of
+                                  Just t -> read $ T.unpack t 
+                                  Nothing -> if isBook || n == 0 then n + 1 else n  
                       headerText <- case filterChild (named "title") e of
-                                        Just t  -> getInlines t
-                                        Nothing -> return mempty
+                                       Just t  -> case maybeAttrValue "supress" t of
+                                                     Just s -> if s == "no" 
+                                                                 then getInlines t 
+                                                                 else return mempty
+                                                     Nothing -> getInlines t
+                                       Nothing -> do
+                                           let name = qName (elName e)
+                                           if (name == "dedication" || name == "foreword" || name == "preface") 
+                                             then return $ str $ T.toTitle name
```

Comment:
It would be nice to use localized (translated) versions of these names rather than the English.
However, we don't currently have localizations for these, so this will have to wait.

Reply:
Good point. The rationale is that, if there are non-English language named sections, then these will probably have a  <title>, captured in the case just above. But might still be something to consider in the future.